ske-operator/values.yaml

```diff
@@ -69,3 +69,10 @@ skeDeployment:
   enabled: true
   # The version to deploy
   version: "latest"
+  tlsConfig:
+    certManagerConfig:
+      disabled: false
```

Comment:
Is the logic the right way around here? I observed errors for the absence of cert manager when this was false but not true.

Is this toggle saying:
- The the use of the provided certManagerConfig is disabled, so use cert manager
or
- The use of the defaultCertManager config is disabled, so do not use cert manager, use the default cert manager config

Reply:
I think neither 😅 

I think perhaps if it was `certManager` not `certManagerConfig` it would be clearer. So `certManager.disabled`  set to `true` means disable it, `false` means dont disable it